### PR TITLE
run black on PR and fix formatting issues in autogenerate_pipeline.py

### PR DESCRIPTION
**Pull request by stefano-garzarella**
### Summary of the PR

The last commit made `black` unhappy.
Let's fix the formatting issues suggested by `black` and run it on every PR to avoid this issue in the future.

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR have Signed-Off-By trailers (with
  `git commit -s`), and the commit message has max 60 characters for the
  summary and max 75 characters for each description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] All added/changed public-facing functionality has entries in the "Upcoming 
  Release" section of CHANGELOG.md (if no such section exists, please create one).
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

**Review by roypat — DISMISSED** [review]

File: .buildkite/autogenerate_pipeline.py [hunk 1]
```diff
@@ -246,7 +247,9 @@ def build(self, input):
             # Wrap command with '' to avoid escaping early by `ENTRYPOINT`
             command = json.dumps(command)
             # Overwrite image tag for riscv64 platform CI
-            self.step_config["plugins"][0][f"docker#{DOCKER_PLUGIN_VERSION}"]["image"] = f"rustvmm/dev:{CONTAINER_VERSION_RISCV}"
+            self.step_config["plugins"][0][f"docker#{DOCKER_PLUGIN_VERSION}"][
+                "image"
+            ] = f"rustvmm/dev:{CONTAINER_VERSION_RISCV}"
```
[the remark this answers is not in the document]
Comment (roypat) [reply .buildkite/autogenerate_pipeline.py:252]: I really hate it when black does this, but oh well haha

**Comment by roypat** [comment]
oh, I just noticed that we actually have a black github action. But why does it run after merge instead of on PRs? :o Should we change https://github.com/rust-vmm/rust-vmm-ci/blob/752ad1322af8a5f632812b1b208b18bdf4e10ce6/.github/workflows/black.yaml#L2 to be `on: pull_request` instead? :thinking:

**Comment by stefano-garzarella** [comment]
> oh, I just noticed that we actually have a black github action. But why does it run after merge instead of on PRs? :o Should we change
> 
> https://github.com/rust-vmm/rust-vmm-ci/blob/752ad1322af8a5f632812b1b208b18bdf4e10ce6/.github/workflows/black.yaml#L2
> to be `on: pull_request` instead? 🤔

I was going to ask, I'm pretty sure I already fixed some other issue in the past years, and maybe asked the same, but didn't find anything.

File: .buildkite/autogenerate_pipeline.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (stefano-garzarella) [reply .buildkite/autogenerate_pipeline.py:252]: me too :-D

[the remark this answers is not in the document]
Comment (stefano-garzarella) [reply .buildkite/autogenerate_pipeline.py:252]: https://github.com/rust-vmm/rust-vmm-ci/pull/141#issuecomment-1774645903 

You complained exactly the same on my old PR!

**Comment by roypat** [comment]
> > oh, I just noticed that we actually have a black github action. But why does it run after merge instead of on PRs? :o Should we change
> > https://github.com/rust-vmm/rust-vmm-ci/blob/752ad1322af8a5f632812b1b208b18bdf4e10ce6/.github/workflows/black.yaml#L2
> > 
> > to be `on: pull_request` instead? 🤔
> 
> I was going to ask, I'm pretty sure I already fixed some other issue in the past years, and maybe asked the same, but didn't find anything.

Ahhh, yeah, now I see. I completely misunderstood you on the other issue last year, sorry! Let's just move this github action to run on PRs instead of post-merge then. I really don't see a good reason why we shouldnt (we check formatting pre-merge on all other repos after all), and it already caused it to have to do fix up PRs at least twice now

**Comment by stefano-garzarella** [comment]
@roypat yeah, I'm going to add a commit to this PR

**Review by roypat — APPROVED** [review]

**Comment by TimePrinciple** [comment]
Should we wait until this is merged, and then upgrade `kvm-bindings`'s CI? I saw the version bump on `kvm-bindings` is about to be merged

**Comment by stefano-garzarella** [comment]
> Should we wait until this is merged, and then upgrade `kvm-bindings`'s CI? I saw the version bump on `kvm-bindings` is about to be merged

This is just a fix for a formatting issue, so IMHO it doesn't really matter.

**Comment by stefano-garzarella** [comment]
Just rebased since our CI on main is still red, @andreeaflorescu @lauralt @sameo @ShadowCurse please, can you take a look?

**Review by ShadowCurse — APPROVED** [review]

**Pull request merged**